src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+
 public class SalaryInfo {
+    private static final int WORK_DAY = 0;
+    private static final int NAME = 1;
+    private static final int WORK_HOURS = 2;
+    private static final int PAY_PER_HOUR = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        Dates dates = new Dates();
```

Comment:
I think in the comment about not putting it in separate classes, Nastia meant all classes